### DISCUSSION TITLE
More flexibility for fields in system collections

### DISCUSSION THREAD

**Discussion by GinMitch** [directus/directus#18222] [Feature Requests]
### Summary

Hi fellow devs,

This feature request relies on probably a tiny amount of Directus' use cases but it's quite important for user experience.

Within our application we have users with a special managerial role that are allowed to create additional users. We've added some fields to the `directus_users` collection and store mandatory (for us) data in them.

The problem is that when a manager creates a new user, they have to scroll through all of the system fields. And there is no way to gracefully adjust their order or even better visibility. For example, we don't utilize the `description` field which takes 3 times more of the vertical space than a "regular" input.

### Motivation

Fields in system collections should be configurable in terms of their visibility or at least their order.

### Detailed Design

Since fields like `tags` and `description` are mandatory for users to access the admin app itself I won't even suggest removing them with permissions.

**Comment by yannbolliger** [comment]
I very much agree. 
It's awesome that Directus provides the users as-they-are. But we'd also like to remove irrelevant fields in our app, like location/title...

**Comment by Pachat** [comment]
On the other hand, I'd like to add some fields. This is usually possible.
But in the case of directus_folders, if I add an input, I don't see where I can feed it.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
